Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -2,7 +2,6 @@ commonfields:
   id: RSA NetWitness Packets and Logs
   version: -1
 name: RSA NetWitness Packets and Logs
-system: true
```

Comment:
Actually no. The implementation already supports it. It makes things much simpler. Playbook, script, BYOI is system only if it comes from content. All the rest are not system. It just we need to update all the scripts/integrations/playbooks in the content repo